website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -147,7 +149,13 @@ You can trigger a test notification from the workspace notifcation settings page
 
 ### Workspace Notification Payload
 
-Workspace events include detailed information about workspace-level validation events like [health assessments](/terraform/cloud-docs/workspaces/health) if you enable them for the workspace. Much of the information provides details about the associated [assessment result](/terraform/cloud-docs/api-docs/assessment-results), which Terraform Cloud uses to track instances of continuous validation. Generic notifications for workspace events contain the following information:
+Workspace events include detailed information about workspace-level validation events like [health assessments](/terraform/cloud-docs/workspaces/health) if you enable them for the workspace. Much of the information provides details about the associated [assessment result](/terraform/cloud-docs/api-docs/assessment-results), which Terraform Cloud uses to track instances of continuous validation.
+
+The types of attributes returned in the payload details will vary depending on the type of `trigger_scope`.
+
+#### Health Assessments
+
+Health assessment notifications for workspace events contain the following information:
 
 <!-- BEGIN: TFC:only name:pnp-callout -->
```

Comment:
I think this red highlighting and the rest that follows is some sign that syntax/format is off. But, I can't identify what it is.

Reply:
Unfortunately, this syntax highlighting is because we are using a callout and for some reason the HTML comment syntax makes MDX mad. I keep meaning to look this, but haven't gotten a chance yet 😢